tests/PhakeNamedArgumentsTest.php

```diff
@@ -0,0 +1,165 @@
+<?php
```

Comment:
I don't think I have ever seen this before. Clever.

Reply:
There is probably a better way to do it using Phpunit feature ? But this was the first thing that came to my mind at that point